env.sample

```diff
@@ -1 +1,10 @@
-POSTGRES_PASSWORD=postgrespassword
\ No newline at end of file
+POSTGRES_PASSWORD=postgrespassword
+LANTERN_ENDPTMGR_DBHOST=localhost
```

Comment:
The .env file is for environment variables that are going to be used in the docker-compose file. The only environment variable currently used in the docker-compose file is $POSTGRES_PASSWORD